hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/commit/ITestAbfsRenameStageFailure.java

```diff
@@ -41,6 +42,12 @@ public ITestAbfsRenameStageFailure() throws Exception {
     binding = new ABFSContractTestBinding();
   }
 
+  protected boolean isNamespaceEnabled() {
+    FileSystem fs = getFileSystem();
+    String namespaceEnabled = fs.getConf().get("fs.azure.test.namespace.enabled");
```

Comment:
there's a Configuration.getBoolean(key, defval) which should be used for evaluating true/false. it knows about trimming strings, case independence etc. have a look at the class and see what else it can do. using the basic get() without a default is generally a rare action.

better.

```
return conf.getBoolean("fs.azure.test.namespace.enabled", false)
```

2. other tests, specificlly everything under `AbstractAbfsIntegrationTest`, probe the fs for having ACLs and use that to dynamically determine fs capabilities. see `getIsNamespaceEnabled()` for how it is done.

i would prefer you copy the AbstractAbfsIntegrationTest algorithm as it is based on the actual FS capabilities

Reply:
Have made changes accordingly in the latest commit (picking up file capabilities dynamically instead of from the test config). Can you please take a look at it? Thank you!